### PR TITLE
Removed log redundancy in QRCode activity

### PR DESCRIPTION
**Pull request by Scar26**
prevented the same word from being logged multiple times as mentioned in #314

### CODE REVIEW

**Comment by Scar26** [comment]
Done @llaske

File: activities/QRCode.activity/js/activity.js [hunk 1]
```diff
@@ -210,9 +210,11 @@ define(["sugar-web/activity/activity","sugar-web/datastore", "sugar-web/env", "w
 
 		// Add entry to history
 		function addToHistory(text) {
+			if(!history.includes(text)){
 			history.push(text);
 			updateHistory();
 		}
+		}
```
Comment (llaske) [inline activities/QRCode.activity/js/activity.js]: Could you realign code to respect indentation?

Comment (Scar26) [reply activities/QRCode.activity/js/activity.js]: done @llaske

Comment (llaske) [reply activities/QRCode.activity/js/activity.js]: No you don't. Of course, line 213 and 214 must be aligned with block.

Comment (Scar26) [reply activities/QRCode.activity/js/activity.js]: Is it fine now? @llaske

**Comment by llaske** [comment]
Ok thanks. It fix the issue.

BTW the history content is case insensitive but QR code are case sensitive.
So it's a bug to consider history as case sensitive. It should be nice to fill a bug on this.

**Pull request merged**